### DISCUSSION TITLE
backup-ed whole $HOME/.chia， but didn't save the key mnemonic, can I get back my key?

### DISCUSSION THREAD

**Discussion by grizzlybears** [Chia-Network/chia-blockchain#4890] [Support]
I had a machine running chia, but now the machine was gone.
I had the backup of the whole $HOME/.chia， but unfortunately I didn't save the key mnemonic.

Now I installed chia on a new machine, and restore the 【old $HOME/.chia】 to $HOME，
when I 'chi init'， I got 

```
(venv) XXXX@fedora chia-blockchain (latest) $ chia init

Chia directory /home/XXXX/sda5/.chia/mainnet
No keys are present in the keychain. Generate them with 'chia keys generate'
/home/shaozr/XXXXX/.chia/mainnet already exists, no migration action taken
```
Do I have chance to get back my key?

**Comment by quimnut** [comment]
try ~/.local too

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
